playground/tests/conftest.py

```diff
@@ -0,0 +1,10 @@
+import pytest_container
+import os
+
+TW = pytest_container.Container(
+    url=f"containers-storage:{os.environ['REGISTRY']}/{os.environ['IMAGE_NAME']}",
```

Comment:
The `containers-storage:` prefix implies that the image is already present in the local image store and will **not** be pulled from the registry. So unless you want to manually pull it from the registry, I would suggest to drop that prefix:

```suggestion
    url=f"{os.environ['REGISTRY']}/{os.environ['IMAGE_NAME']}",
```

Reply:
The image should be present in the local image store. I can't figure out why it is not there.

Reply:
Why should it be there? You build it with docker and then run the tests with podman. podman will not look into docker's store when the `containers-storage:` prefix is used (that's for consistency with upstream behavior).

Why do you want to use the locally build image at all cost? Imho it only complicates matters for you.

Reply:
@dcermak I'm starting to understand what you are talking about. :-) 
The idea is to build and test the image in CI before it is pushed to a registry.  AFAIK, it is impossible to push the image to ghcr.io from a forked repository PR. That is why I'm trying to pick it up locally. Does it make sense?

Reply:
Switched the build step to buildah and it looks better now.  Your insight helped me figure this out. Thank you.

Reply:
Happy to help, thanks for using pytest_container!